src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -212,14 +211,23 @@ public void RemoveFunction(string name)
         {
             Inc();
 
-            _functions.RemoveAll(func => func.Name == name);
+            _functions.RemoveAll(name);
         }
 
         internal void RemoveFunction(TexlFunction function)
         {
             Inc();
 
-            _functions.RemoveAll(func => func == function);
+            _functions.RemoveAll(function);
+        }
+
+        internal void AddFunctions(TexlFunctionSet functions)
+        {
+            Inc();
+            _functions.Add(functions);
+
+            // Add any associated enums 
+            EnumStoreBuilder?.WithRequiredEnums(functions);
         }
 
         internal void AddFunction(TexlFunction function)
```

Comment:
>AddFunction [](http://example.com/codeflow?start=22&length=11)

Can this just call the above? 

AddFunctions(new TexlFunctionSet(function))

Reply:
They look very similar but are really executing very different code under the hood

Reply:
These two methods are very different